pretix_eth/static/pretix_eth/question-type.js

```diff
@@ -1,7 +1,62 @@
+"use strict";
+
 /*
  * payment_eth_html_add_question_type_javascript
  */
 
+// Unpkg imports
+const Web3Modal = window.Web3Modal.default;
+const WalletConnectProvider = window.WalletConnectProvider.default;
+const EvmChains = window.EvmChains;
+
+// Web3modal instance
+let web3Modal;
+
+// Chosen wallet provider given by the dialog window
+let provider;
+
+// Address of the selected account
+let selectedAccount;
+
+function init() {
+  const providerOptions = {
+    walletconnect: {
+      package: WalletConnectProvider,
+    },
+  };
+
+  web3Modal = new Web3Modal({
+    cacheProvider: false, // optional
+    providerOptions, // required
+  });
+}
+
+async function refreshAccountData() {
+  const web3 = new Web3(provider);
+  const accounts = await web3.eth.getAccounts();
+  const inputElements = document.querySelectorAll('input'); 
```

Comment:
So if we add more inputs/questions to this form, this will get screwed up? We're definitely planning on adding more...

Is there a way to add an ID to this input and select it based on that? Maybe Jakub or Ligi would know if Pretix allows that when adding a question?